proc_scripts/load_data.py

```diff
@@ -52,27 +102,134 @@ def proc_spincore_CPMG_v1(s, fl=None):
+    # {{{ these are hard-coded for the pulse sequence
+    #     if we need to, we could pull these from the pulse sequence, as we do
+    #     for anavpt above
+    tau_extra = 20e-6
+    tau_pad_start = tau_extra-dwdel1-6e-6
+    tau_pad_end = tau_extra-6e-6
+    twice_tau = 2*p90_s + 5e-6 + tau_pad_start + 1e-6 + acq_time + tau_pad_end +1e-6
+    #twice_tau should be the period from one 180 to another
+    # }}}
+    s.set_units('t2','us')
+    s.chunk('t2',['tE','t2'],[nEchoes,-1])
+    s.setaxis('tE', (1+r_[0:nEchoes])*twice_tau)
+    s.ft('t2', shift=True).ft(['ph1','ph2'])
+    s.reorder(['ph1','ph2','indirect'])
+    if fl is not None:
+        fl.next('freq domain coh domain')
+        fl.image(s)
+    s.ift('t2')
+    if fl is not None:
+        fl.next('t2 chunked', figsize=(5,20))
+        fl.image(s)
+    s.ft('t2')
+    return s
+
+def proc_bruker_CPMG_v1(s,d12,tau_extra,fl=None):
+    logger.info(strm("pre-processing CPMG acquired on Bruker"))
+    s.chunk('indirect',['ph1','ph2','indirect'],[4,2,-1])
+    s.setaxis('ph1',r_[0,4]/4.)
+    s.setaxis('ph2',r_[0,2]/2.)
+    if fl is not None:
+        fl.next('raw data before')
+        fl.image(s)
+    s.ft(['ph1','ph2'])
+    s.reorder(['ph1','ph2','indirect','t2'])
+    anavpt_info = [j for j in s.get_prop('pulprog').split('\n') if 'anavpt' in j.lower()]
+    anavpt_re = re.compile(r'.*\banavpt *= *([0-9]+)')
+    anavpt_matches = (anavpt_re.match(j) for j in anavpt_info)
+    for m in anavpt_matches:
+        if m is not None:
+            anavpt = int(m.groups()[0])
+    actual_SW = 20e6/anavpt # JF: check that this is based on the manual's definition of anavpt
+    bruker_final_t2_value = double(s.getaxis('t2')[-1].item())
+    s.setaxis('t2',1./actual_SW*r_[0:ndshape(s)['t2']]) # reset t2 axis to true values based on anavpt
+    logger.debug(strm("the final t2 value according to the Bruker SW_h was",
+        bruker_final_t2_value, "but I determine it to be",
+        double(s.getaxis('t2')[-1].item()), "with anavpt"))
+    nEchoes = s.get_prop('acq')['L'][25]
+    dwdel1 = s.get_prop('acq')['DE']*1e-6
+    dwdel2 = (anavpt*0.05e-6)/2
+    #d12 is read as 0 if taken from parameters bc its too small
+    d11 = s.get_prop('acq')['D'][11]
+    p90_s = s.get_prop('acq')['P'][1]*1e-6
+    quad_pts = ndshape(s)['t2'] # note that we have not yet chunked t2
+    nPoints = quad_pts/nEchoes
+    acq_time = dwdel2*nPoints*2
+    # {{{ these are hard-coded for the pulse sequence
+    #     if we need to, we could pull these from the pulse sequence, as we do
+    #     for anavpt above
+    tau_pad_start = tau_extra-dwdel1-6e-6
+    tau_pad_end = tau_extra-6e-6
+    twice_tau = 2*p90_s + 5e-6 + tau_pad_start + 1e-6 + acq_time + tau_pad_end +1e-6
+    #twice_tau should be the period from one 180 to another
+    # }}}
+    assert twice_tau = dwdel1 + 5e-6 +tau_pad_start + 1e-6 + dwdel2*nPoints + tau_pad_end + 1e-6 + 5e-6, "twice_tau is not equal to the time from one 180 pulse to the next" 
```

Comment:
does this even run? you would need a `==` not a `=`.  Just see my most recent comment in [this thread](https://github.com/jmfrancklab/proc_scripts/pull/34/files#r559809273), which I think should clear up any remaining confusion + get us on our way!

Reply:
works on my end now